### PR TITLE
added reference to hunting in MDATP

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2312

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @yogkumgit, for your information

Thank you

File: microsoft-365/security/mtp/advanced-hunting-overview.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Advanced hunting is a query-based threat-hunting tool that lets you explore up t
 
 You can use the same threat-hunting queries to build custom detection rules. These rules run automatically to check for and respond to various events and system states, including suspected breach activity and misconfigured machines.
 
-In the Microsoft 365 security center, advanced hunting supports queries that look into data from various workspaces, including data about devices, emails, apps, and identities from Microsoft Defender ATP, Office 365 ATP, Microsoft Cloud App Security, and Azure ATP. To use advanced hunting, [turn on Microsoft Threat Protection](mtp-enable.md).
+The feature is similar to the [advanced hunting in Microsoft Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-overview), except that in the Microsoft 365 security center, advanced hunting supports queries that look into data from various workspaces, including data about devices, emails, apps, and identities from Microsoft Defender ATP, Office 365 ATP, Microsoft Cloud App Security, and Azure ATP. To use advanced hunting, [turn on Microsoft Threat Protection](mtp-enable.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/advanced-hunting-overview.md:34]:
```suggestion
The feature is similar to the [advanced hunting in Microsoft Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-overview), except that in the Microsoft 365 security center, advanced hunting supports queries that look into data from various workspaces, including data about devices, emails, apps, and identities from Microsoft Defender ATP, Office 365 ATP, Microsoft Cloud App Security, and Azure ATP. To use advanced hunting, [turn on Microsoft Threat Protection](mtp-enable.md).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lomayor Please merge this PR. Thanks.

File: microsoft-365/security/mtp/advanced-hunting-overview.md (hunk 1 above)
Comment (lomayor) [inline microsoft-365/security/mtp/advanced-hunting-overview.md]: Remove "the" before publishing:

The feature is similar to **the** [advanced hunting in Microsoft Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-overview), except...

Comment (lomayor) [reply microsoft-365/security/mtp/advanced-hunting-overview.md]: @yogkumgit, please apply the minor change before I merge.

Comment (MaratMussabekov) [reply microsoft-365/security/mtp/advanced-hunting-overview.md]: Dear @lomayor ,
done

**Pull request merged**